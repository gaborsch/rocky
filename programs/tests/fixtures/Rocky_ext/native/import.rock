from java, math play BigDecimal, BigInteger
from java, io play Serializable
say "Import OK"

Rock my array with "Import", "Turns", "On", "Extension", "Mode"
say my array sorted
say "Import turned on extension mode"

say "BigDecimal is " + BigDecimal
say "BigInteger is " + BigInteger
say "Serializable is " + Serializable
say "Static instance created"