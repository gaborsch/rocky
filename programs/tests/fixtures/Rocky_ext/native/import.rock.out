Import OK
5
Import turned on extension mode
BigDecimal is java.math.BigDecimal
BigInteger is java.math.BigInteger
Serializable is java.io.Serializable
Static instance created
